components/admin_console/admin_sidebar/admin_sidebar.test.jsx

```diff
@@ -5,8 +5,13 @@ import React from 'react';
 import {IntlProvider} from 'react-intl';
 import {shallow} from 'enzyme';
 
+import {samplePlugin1} from '../../../tests/helpers/admin_console_plugin_index_sample_pluings';
```

Comment:
same with this `tests/helpers/admin_console_plugin_index_sample_pluings'` should suffice i think 🤔